__tests__/utils.ts

```diff
@@ -55,6 +59,7 @@ export const webhookBodyFields: {
 		},
 	},
 	status: {
+		errors: [webhookError, webhookError],
```

Comment:
Why is the err added twice?

Reply:
Only because the errors property under `changes[0].value.errors` also contains the error twice.

Reply:
I changed it to one entry